tests/test_registrations/test_review_flows.py

```diff
@@ -135,7 +135,6 @@ def test_rejection_flow(self, sanction_object, initial_state, end_state):
 
     @pytest.mark.parametrize('sanction_object', [registration_approval, embargo, retraction])
     def test_approve_after_reject_fails(self, sanction_object):
-        # using fixtures in parametrize returns the function
```

Comment:
It's not a huge deal, but if you could go through the file and delete all iterations of this comment, that would be appreciated